DFReader.py

```diff
@@ -605,8 +617,8 @@ class DFReader_text(DFReader):
     def __init__(self, filename, zero_time_base=False):
         DFReader.__init__(self)
         # read the whole file into memory for simplicity
-        f = open(filename, mode='r')
-        self.lines = f.readlines()
+        with open(filename, 'r') as f:
+            self.lines = f.readlines()
         f.close()
```

Comment:
Running the same log through with python2 vs python3 is giving different results:
```
-2016-11-02 01:11:40.72: LOCAL_POSITION_NED {time_boot_ms : 350433, x : 1.2293812036514282, y : 0.10330259054899216, z : -30.021739959716797, vx : -0.19804619252681732, vy : 0.17870385944843292, vz : -0.5509760975837708}
+2016-11-02 01:11:40.72: LOCAL_POSITION_NED {time_boot_ms : 350433, x : 1.22938120365, y : 0.103302590549, z : -30.0217399597, vx : -0.198046192527, vy : 0.178703859448, vz : -0.550976097584}
```